mobile/native/src/calculations/mod.rs

```diff
@@ -31,3 +32,13 @@ pub fn calculate_liquidation_price(price: f64, leverage: f64, direction: Directi
 
     liquidation_price
 }
+
+pub fn calculate_fees() -> u64 {
```

Comment:
:wrench: I think the transaction fees should be calculated by the ln-dl-node crate.